packages/overlay/src/integrations/sentry/sentry-integration.ts

```diff
@@ -1,42 +1,27 @@
 import type { Client, Envelope, Event, Integration } from '@sentry/types';
 import { serializeEnvelope } from '@sentry/utils';
-import { DEFAULT_SIDECAR_URL } from '~/constants';
+import { trigger } from '../../lib/eventTarget';
 import { log } from '../../lib/logger';
 import sentryDataCache from './data/sentryDataCache';
-import { getNativeFetchImplementation } from './utils/fetch';
-
-type SpotlightBrowserIntegrationOptions = {
-  /**
-   * The URL of the Sidecar instance to connect and forward events to.
-   * If not set, Spotlight will try to connect to the Sidecar running on DEFAULT_SIDECAR_URL.
-   *
-   * @default DEFAULT_SIDECAR_URL
-   */
-  sidecarUrl?: string;
-};
 
 /**
  * A Sentry integration for Spotlight integration that the Overlay will inject automatically.
  * This integration does the following:
  *
  *  - Drop transactions created from interactions with the Spotlight UI
  *  - Forward Sentry events sent from the browser SDK to the Sidecar instance running on
- *    either on http://localhost:8969/stream or on the supplied `sidecarUrl` option.
- *
- * @param options - Configuration options for the integration ({@link SpotlightBrowserIntegrationOptions})
+ *    the same page via the "direct message" method (w/o a need for the sidecar)
  *
  * @returns Sentry integration for Spotlight.
  */
-export const spotlightIntegration = (options?: SpotlightBrowserIntegrationOptions) => {
-  const _sidecarUrl = options?.sidecarUrl ?? DEFAULT_SIDECAR_URL;
-
+export const spotlightIntegration = () => {
```

Comment:
Do we export this from the overlay package? I don't think so but I'm just checking, since this is a breaking change if it was public API.

Reply:
Yeah I'm aware. I'm fine as it won't really break if people pass in an options object, it will just be ignored which should be fine in this specific scenario.

Reply:
Possibly. But again, this is not meant to be used externally so 🤷🏻‍♂️